### PR TITLE
Run task accepts varargs

### PR DESCRIPTION
**Pull request by Invertisment**

### CODE REVIEW

File: packages/builder/src/internal/core/errors-list.ts [hunk 1]
```diff
@@ -452,21 +452,21 @@ Please double check how you invoked algob or run your task.`,
...
   BUILTIN_TASKS: {
-    RUN_FILE_NOT_FOUND: {
+    RUN_FILES_NOT_FOUND: {
       number: 601,
-      message: "Script %script% doesn't exist.",
-      title: "Script doesn't exist",
-      description: `Tried to use \`algob run\` to execute a non-existing script.
+      message: "Scripts don't exist: %scripts%.",
+      title: "Scripts don't exist.",
+      description: `Tried to use \`algob run\` to execute a non-existing script(s).
```
Comment (robert-zaremba) [inline packages/builder/src/internal/core/errors-list.ts:459]: this message doesn't make much sense. A user is already running `algob run`. 
Also, why we are removing a filename from the message?

File: packages/builder/test/helpers/project.ts [hunk 2]
```diff
@@ -54,3 +60,22 @@ export function useFixtureProjectCopy(srcProjectName: string) {
...
+  });
+
+  beforeEach(function () {
+    try {
+      fs.unlinkSync(testFixtureOutputFile)
+    } catch (err) {
+      // ignored
+    }
+  })
+}
```
Comment (robert-zaremba) [inline packages/builder/test/helpers/project.ts:81]: Isn't this a copy-paste from some task test? (eg TASK_CLEAN)? If yes, then let's remove the function form that test.

Comment (Invertisment) [reply packages/builder/test/helpers/project.ts:81]: I moved this to here from deploy tests.

File: packages/builder/src/internal/core/errors-list.ts (hunk 1 above)
Comment (Invertisment) [reply packages/builder/src/internal/core/errors-list.ts:459]: This message appears when user specifies the scripts that he wants to execute:
`algob run 1 2 3`.
When any of those don't exist they are displayed in this message.
If scripts 2 and 3 don't exist they will be displayed this way:
`Scripts don't exist: 1,2`

Comment (robert-zaremba) [reply packages/builder/src/internal/core/errors-list.ts:459]: OK. Good.
We shall abort running all scripts, when any of them doesn't exists.

**Comment by Invertisment** [comment]
> Looks good. Please confirm that we first verify if all scripts exist before starting the execution.

File checks are done only for run. And they are done only before the execution.
There is no point to do it for deploy because user doesn't specify them.

**Pull request merged**